### PR TITLE
Rewrite1

### PR DESCRIPTION
**Pull request by tom6931**
Rewrite of documentation.  Highlights:

- include suggestions from CSD testing into quickstarts.
- Changed quickstart examples to better reflect real-world example.
- added tutorials sections with some other examples.
- Modified overview per CSD suggestions

### CODE REVIEW

File: docs/source/quickstart-aws.rst [hunk 1]
```diff
@@ -44,61 +51,34 @@ Get ElasticBLAST
...
-#. :ref:`query sequences <elb_queries>`,
-
-#. a :ref:`cloud storage bucket for results <elb_results>`. This value must start with ``s3://`` and _uniquely_ identifies your ElasticBLAST search. **Please keep track of this**.
-
-#. basic BLAST parameters (:ref:`program <elb_blast_program>` and :ref:`database <elb_db>`), and
-
-#. :ref:`elb_num_nodes` to start.
+You will use a configuration file to specify your input to ElasticBLAST.  Once you have written the configuration file, you'll just need to tell ElasticBLAST about it when invoked.
 
+Start by, copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini".  Vi is pre-installed in the CloudShell.  Instructions for installing nano on the CloudShell can be found `here <https://docs.aws.amazon.com/cloudshell/latest/userguide/vm-specs.html#installing-software>`__.
```
Comment (christiam) [inline docs/source/quickstart-aws.rst:79]:
```suggestion
Start by, copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini". `Vi <https://en.wikipedia.org/wiki/Vi>`_ is pre-installed in the CloudShell.  Instructions for installing nano on the CloudShell can be found `here <https://docs.aws.amazon.com/cloudshell/latest/userguide/vm-specs.html#installing-software>`__.
```

File: docs/source/quickstart-gcp.rst [hunk 2]
```diff
@@ -58,21 +76,9 @@ Set up an output bucket (if one doesn't exist)
...
+You will use a configuration file to specify your input to ElasticBLAST.  Once you have written the configuration file, you'll just need to tell ElasticBLAST about it when invoked.
 
-#. basic BLAST parameters (:ref:`program <elb_blast_program>` and :ref:`database <elb_db>`), and
-
-#. :ref:`elb_num_nodes` to start.
-
-
-
-They can be provided on a standard `ini configuration file format <https://en.wikipedia.org/wiki/INI_file>`_, e.g.:
+Start by copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini".  Both nano and vi are available on the Cloud Shell.
```
Comment (christiam) [inline docs/source/quickstart-gcp.rst:81]:
```suggestion
Start by copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini".  Both `nano <https://www.nano-editor.org/>`_ and `vi <https://en.wikipedia.org/wiki/Vi>`_ are available on the Cloud Shell.
```

File: docs/source/quickstart-aws.rst (hunk 1 above)
Comment (tom6931) [reply docs/source/quickstart-aws.rst:79]: I don't think we need a wikipedia link for vi.  If folks don't know what it it, they can google it.  For most users it becomes a link you click on and then realize you didn't need it.

File: docs/source/quickstart-gcp.rst (hunk 2 above)
Comment (tom6931) [reply docs/source/quickstart-gcp.rst:81]: Same comment as above.  If folks don't know what it is, they can google it.   The link is distracting.

**Pull request merged**